### PR TITLE
Bunch of added things to the risk page

### PR DESCRIPTION
**Pull request by Pato-desu**
I think some sections can be a lot more subjective that what I've seen on the website or that at least I haven't seen other people talk about them (probably because they are not as important as x-risks), so others could not agree with them, so you should read them closely.

Also it could be written not very well and with a bunch of grammatical errors, so feel free to change it as you would prefer. I think it's better to have them amateurish written (and without links) than to not have them at all.

### CODE REVIEW

**Review by joepio — CHANGES_REQUESTED** [review]

File: src/posts/risks.md [hunk 1]
```diff
@@ -21,16 +23,22 @@ We might soon see social media be flooded with fake discussions and opinions, an
 This could erode the trust we have in our society.
 It could threaten the fundamentals of our democracy even more than social media did.
 
-## Biases and discrimination
+### Deepfakes and impersonification
+
+Deepafakes not only can steal famous people identities and create disinformation, but can impersonate you too. 
+Anyone with photos, videos, or audios of someone and enough knowledge, can create deepfakes of them (soon in real time and with high quality) and use them to commit fraud, harass them, or create sexually nonconsensual material (which it's already common).
```
[the remark this answers is not in the document]
Comment (Pato-desu) [reply src/posts/risks.md:29]: I'll add that link, and yeah, we could add sources everywhere, in each section. right now i'm kinda busy

File: src/posts/risks.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ description: AI threatens our democracy, our technology, and our species.
 AI is a powerful technology that is increasingly transforming our world.
 It comes with amazing potential, but also with serious risks, including [existential catastrophe](/xrisk).
 
-## Fake media
+## Present dangers
```
[the remark this answers is not in the document]
Comment (Pato-desu) [reply src/posts/risks.md:9]: I think it's pretty self-explanatory, feel free to add one if you want though

File: src/posts/risks.md [hunk 3]
```diff
@@ -6,7 +6,9 @@ description: AI threatens our democracy, our technology, and our species.
...
+
+### Fake news, polarization and democracy
```
[the remark this answers is not in the document]
Comment (Pato-desu) [reply src/posts/risks.md:11]: I understand what you mean, I don't think I wrote about polarization, but really fake social media content and fake news should go hand and hand together. It's not that people see stuff that they don't know if it's true or and not knowing what it's true make them want to overcome democracy, like this section seems to imply. It's that fake info could lead to distrust of institutions that could lead to people wanting to overcome democracy.

File: src/posts/risks.md [hunk 4]
```diff
@@ -91,7 +120,16 @@ Instead of having to hire a team of skilled security experts/hackers to find zer
 
 [Read more about AI and cybersecurity risks](/cybersecurity-risks)
 
-## Existential Risk
+### Human disempowerment
+
+Whether AIs centralize power around certain actors or not, and even if single systems don't optimize strongly for any particular measure that we cannot steer or fix, we could lose our power to take decisions incrementally with each AI system that gets implemented inside institutions or gets popular on everyday life.
+
+Those systems would have more input from other systems than from humans, and, if we cannot coordinate quick enough or we lack crucial knowledge about the functioning of these systems, we could end up inside a civilization in which each system is optimizing for different things, there is not a clear direction for where everything is heading, and there is not any way of changing it.
```
[the remark this answers is not in the document]
Comment (Pato-desu) [reply src/posts/risks.md]: yeah, it's complicated to explain disempowerment. check the changes that I did now

File: src/posts/risks.md [hunk 5]
```diff
@@ -21,16 +23,22 @@ We might soon see social media be flooded with fake discussions and opinions, an
 This could erode the trust we have in our society.
 It could threaten the fundamentals of our democracy even more than social media did.
 
-## Biases and discrimination
+### Deepfakes and impersonification
+
+Deepafakes not only can steal famous people identities and create disinformation, but can impersonate you too.
```
[the remark this answers is not in the document]
Comment (Pato-desu) [reply src/posts/risks.md]: done

File: src/posts/risks.md [hunk 6]
```diff
@@ -91,7 +120,16 @@ Instead of having to hire a team of skilled security experts/hackers to find zer
...
+
+Whether AIs centralize power around certain actors or not, and even if single systems don't optimize strongly for any particular measure that we cannot steer or fix, we could lose our power to take decisions incrementally with each AI system that gets implemented inside institutions or gets popular on everyday life.
+
+Those systems would have more input from other systems than from humans, and, if we cannot coordinate quick enough or we lack crucial knowledge about the functioning of these systems, we could end up inside a civilization in which each system is optimizing for different things, there is not a clear direction for where everything is heading, and there is not any way of changing it.
+The technical knowledge to change these systems could be lacking in the first place, or it could be lost in time, as we become more and more dependent on technology, and the technology becomes more complex.
+
+It could even happen that the measurements that the systems care about get increased, but they would not be fully encompassing the actual values that they tried to encapture. This problem, to a certain extent, is already happenning today, but AIs could highly amplify it.
+
```
[the remark this answers is not in the document]
Comment (Pato-desu) [reply src/posts/risks.md]: I ended up putting x-risks above because it's simpler and easier to explain the other stuff afterwards kinda

**Comment by Pato-desu** [comment]
welp, I made some changes. Tell me if they are not enough

**Review by joepio — APPROVED** [review]

**Comment by joepio** [comment]
Great, thanks!

**Pull request merged**